lib/meson.build

```diff
@@ -119,6 +119,7 @@ hse_dependencies = [
     crc32c_dep,
     xoroshiro_dep,
     libpmem_dep,
+    tcmalloc_dep,
```

Comment:
Does this force a dependency on tcmalloc?  What if tcmalloc isn't installed on the machine?